### ISSUE TITLE
Karaoke Plugin

### ISSUE THREAD

**Issue by SuperIchi** [goatcorp/suggestions#794]
### Description
Anytime a song is playing in Final Fantasy XIV that has lyrics, have the lyrics on screen like it's karaoke, so players can sing along. Would make battles a lot more fun, I think..

### More info
Would need to be able to detect what part of the song it's at so it shows the correct verses.

----

 - [] I checked if this request wasn't already proposed
 - [] I made sure this request follows Dalamud's [plugin guidelines](https://github.com/goatcorp/FFXIVQuickLauncher)

**Comment by TheGokki** [comment]
Agree, would be nice, especially in boss fights to see the lyrics as i fight the boss

**Comment by thisischrys** [comment]
Surely not "Anytime a song is playing that has lyrics"? Isn't Close in the Distance playing in Ultima Thule depending on if you finished 6.0 or not? Maybe have a blacklist.

**Comment by ArkfrostLumas** [comment]
Nope, Close in the Distance only plays when you reach the part after the Omicrons. Before that it plays an entirely different song & music file: Close in the Distance (Instrumental) and when you first reach UT it plays Echoes in the Distance.

**Comment by ArkfrostLumas** [comment]
Yes, once you clear the Omicron section Close in the Distance plays in UT forever. Before that Close in the Distance (Instrumental) plays, starting from the Ea area, and before that Echoes in the Distance is playing.
